queries.sql

```diff
@@ -48,4 +48,52 @@ UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg<0;
  SELECT species.name,COUNT(animals.name) FROM animals JOIN species ON animals.species_id=species.id GROUP BY species.name;
  SELECT animals.name, species.name, owners.full_name FROM animals JOIN species ON animals.species_id = species.id JOIN owners ON animals.owner_id = owners.id WHERE species.name = 'Digimon' AND owners.full_name = 'Jennifer Orwell';
  SELECT animals.name,owners.full_name FROM animals JOIN owners ON animals.owner_id=owners.id WHERE owners.full_name='Dean Winchester' AND animals.escape_attempts=0;
- SELECT owners.full_name, COUNT(animals.name) AS num_of_animals FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY num_of_animals DESC LIMIT 1;
\ No newline at end of file
+ SELECT owners.full_name, COUNT(animals.name) AS num_of_animals FROM animals JOIN owners ON animals.owner_id = owners.id GROUP BY owners.full_name ORDER BY num_of_animals DESC LIMIT 1;
+
+SELECT animals.name FROM visits JOIN animals ON visits.animal_id=animals.id 
+ JOIN vets ON visits.vet_id=vets.id WHERE vets.name='William Tatcher' ORDER BY (visits.date_of_the_visit) DESC LIMIT 1;
+
+ SELECT COUNT(*) FROM animals A
+JOIN visits V ON A.id = V.animal_id
+WHERE V.vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
```

Comment:
- [ ] Nice work writing the query for the question (How many different animals did Stephanie Mendez see?) :clap: However we are looking for the number of **different** animals that Stephanie Mendez saw not the total of animals that he/she has seen. Therefore it would be more appropriate to use **DISTINCT** in this case rather that **COUNT**.
**HIT**
```sql
SELECT DISTINCT A.name FROM animals A
JOIN visits V ON A.id = V.animal_id
WHERE V.vet_id = (SELECT id FROM vets WHERE name = 'Stephanie Mendez');
```

Reply:
#### I was able to add different animals query that Mendez saw.Here is the screenshot
![dfrntanimals](https://github.com/DianaBeki/vet_clinic/assets/99320947/dfc0b891-d8a4-4988-bf86-55727ba74a21)